src/typecheck.c

```diff
@@ -246,8 +245,6 @@ static Signature handle_signature(FileTypes *ft, const AstSignature *astsig, con
     else
         sig.returntype = type_from_ast(ft, &astsig->returntype);
 
-    // TODO: validate main() parameters
-    // TODO: test main() taking parameters
```

Comment:
We could keep these todo comments until #257 is fixed.

Reply:
Really? The self-hosted compiler can get args now

Reply:
Good point. We should still add validation: if main has parameters, there must be 2 of them, of types `int` and `byte**`.